.eleventy.js

```diff
@@ -213,6 +216,7 @@ module.exports = function(config) {
   config.addPairedShortcode('CompareCaption', CompareCaption);
   config.addPairedShortcode('Details', Details);
   config.addPairedShortcode('DetailsSummary', DetailsSummary);
+  config.addShortcode('EventTable', EventTable);
```

Comment:
Would you mind adding a comment that we should remove this shortcode if we take down the live page, or if the live page no longer uses it. I think the post-event mocks don't use it so at that point it'd just be dead weight.